src/main/kotlin/org/malachite/estella/people/api/UserController.kt

```diff
@@ -35,6 +74,14 @@ class UserController(@Autowired private val userService: UserService) {
         return ResponseEntity.created(URI("/api/users/" + saved.id)).build()
     }
 
+    @CrossOrigin
+    @GetMapping("/{userId}")
+    fun getUser(@PathVariable userId: Int): ResponseEntity<User> {
+        val user: User = userService.getUser(userId)
+
+        return ResponseEntity(user, HttpStatus.OK)
+    }
+
     @CrossOrigin
     @PutMapping("/{userId}")
```

Comment:
Ten put ma tak działać?

Reply:
Nie wiem @tomros766 to pisał nie zagłębiałem się aż tak :>

Reply:
jak się nietrudno domyślić, jest to standardowe zachowanie tutorialowe xD jak będziemy potrzebowali innej funkcjonalności, to poprawimy

Reply:
Znaczy ogólnie pytam, bo ten put, mimo iż funkcja nazywa się update, to jedyne, co robi, to wyrzuca usera, a z tego, co pamiętam z baz, update tak nie działa XDDDD

Reply:
Robi update, musisz se rozwinąć plik bo git robi collapse jak jest duży kawałek niezmienionego kodu xD

Reply:
Fajnie to złożyło XDDDD